### PR TITLE
Update max-width for site grid

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Update site max-width and adjust header content to it

Screenshot 2024-05-31 at 10 23 32

## Justify why they are needed
We don't want the site to be too wide on larger screens and align with the new UI kit updates

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4315** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4315?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/Header/Header.css.ts [hunk 1]
```diff
@@ -48,21 +49,29 @@ export const ghostWrapper = style({
...
+  },
+})
+
+export const contentWrapper = style({
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'space-between',
-
-  top: 0,
-  zIndex: zIndexes.header,
```
Comment (alebedev) [inline apps/store/src/components/Header/Header.css.ts:57]: Does it work fine without zIndex?

**Review by alebedev — APPROVED** [review]

**Review by guilhermespopolin — APPROVED** [review]
💅🏻

File: apps/store/src/components/Header/Header.css.ts (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/Header/Header.css.ts:57]: It was just moved to code above 👍

**Comment by gustaveen** [comment]
### Merge activity

* **Jun 3, 4:51 AM EDT**: @gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4315).

**Pull request merged**